src/game/common/globaldata.h

```diff
@@ -311,7 +311,7 @@ class GlobalData : public SubsystemInterface
     float m_voiceVolumeFactor; // not 100% sure, needs confirming
     bool m_sound3DPref; // not 100% sure, needs confirming
     bool m_animateWindows;
-    bool m_setMinVertextBufferSize; // not 100% sure, needs confirming
+    bool m_setMinVertexBufferSize; // not 100% sure, needs confirming
```

Comment:
is this comment still relevant.

Reply:
I don't know and its out of scope for this PR, I just fixed the typo :)